README.md

```diff
@@ -10,20 +10,132 @@
 ## What the Student Already Knows
 A student should already understand the concept of variables, constants, Int, String, Bool, Array, Dictionary and functions but not Tuples.
 
-## Outline/ Notes
+## Using Tuples
 
-* Start with the known- students know how to iterate over arrays. Show an example of iterating over an array. 
-    * Present some real world problem such as, “If you have a dictionary where the keys represent the seasons and the values represent your favorite color associated with that season, is there an easy way to print out a list that for each sentence and color that says “In the {season} my favorite color is {color}.”
-    - If you have a better example feel free to use it, but always aim for a real world example. And you can use that example throughout the rest of your reading and/or use other new examples. 
-    * The answer is tuples and this sets the stage for writing the content that covers your learning objectives above.--- You can iterate over the key-value pairs in a dictionary with a for-in loop. Each item in the dictionary is returned as a (key, value) tuple. Explain how you can decompose the tuple’s members into temporary constants as part of the iteration.
-    
+Swift provides a number of ways to group related values together. One basic way that you've already learned about is arrays. Imagine you have a list of things to buy at the grocery store. This is a list of strings: milk, eggs, bread, and so on. This list of items could be stored in an array of strings. Iterating over each element of that array is easy. If you wanted to print out each item in your shopping list, you could use a `for` loop:
+
+```
+let shoppingList = ["milk", "eggs", "bread"]
+for item in shoppingList {
+    print(item)
```

Comment:
Even though it is self explanatory as to what will print here, I like adding a comment line either below or to the right of the print statement that the student can see and then understand what it is that it is printing.
```
print(item)
// milk
// eggs
// bread
```